### DISCUSSION TITLE
Speaker directory

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#178] [Lunch & Learns]
Would it be useful to have a speaker directory? (Sorry if this has been discussed.)

So instead of putting the bio, headshot, etc. in the issue, we drop a link to the speaker directory?

I'm just thinking we have repeat presenters, as well as podcast guests, lightning talks, and more where we ask for the same info. So if there was a quick reference point, we wouldn't have to worry so much about tracking down data for everyone.

**Comment by tkshill** [comment]
I like it. Could also be templatized if necessary. Do we make it something only we access or can speakers also use it to self insert/update

**Comment by danieltott** [comment]
Seems like we could start out with a pretty simple airtable that is internal only and then try to power it up a bit more - even just having it internal would probably make life easier.
